src/util/std_types.cpp

```diff
@@ -73,6 +79,7 @@ typet struct_union_typet::component_type(
   return c.type();
 }
 
+/// Returns true if the object is a prefix of \param other.
```

Comment:
I don't know what it means for a struct to be a prefix of another struct means? Could this define what a prefix means in this context.